docs/changelogs/v0.27.md

```diff
@@ -6,13 +6,18 @@
 
 - [Overview](#overview)
 - [🔦 Highlights](#-highlights)
+  - [Gateway: support for `/api/v0` is deprecated](#gateway-support-for-apiv0-is-deprecated)
 - [📝 Changelog](#-changelog)
 - [👨‍👩‍👧‍👦 Contributors](#-contributors)
 
 ### Overview
 
 ### 🔦 Highlights
 
+#### Gateway: support for `/api/v0` is deprecated
+
+Support for the API endpoint via the gateway port is deprecated and should not be used. It will be removed in the next version. You can read more in https://github.com/ipfs/kubo/issues/10312.
```

Comment:
```suggestion
Support for exposing the legacy subset of Kubo RPC via the Gateway port is deprecated and should not be used. It will be removed in the next version. You can read more in <https://github.com/ipfs/kubo/issues/10312>.

If you have a legacy software that relies on this behavior, and want to expose parts of `/api/v0` next to `/ipfs`, use reverse-proxy in front of Kubo to mount both Gateway and RPC on the same port. NOTE: exposing RPC to the internet comes with security risk: make sure to specify access control via [API.Authorizations](https://github.com/ipfs/kubo/blob/master/docs/config.md#apiauthorizations).
```